src/content/zh-CN/2019/performance.md

```diff
@@ -0,0 +1,277 @@
+  </a>
+  <div id="fig6-description" class="visually-hidden">每个有效连接类型的FCP分布条形图。4G 快速, 中速, 慢速: 14%, 67%以及19%。 3G: 1%, 38%以及61%. 2G: 0%, 9%, 90%. 慢速 2G: 0%, 1%, 99%.</div>
+  <figcaption id="fig6-caption">图6. 被标记为具有快速、中速或慢速FCP的网站分布，按<abbr title="effective connection type">ECT</abbr>分类。</figcaption>
+</figure>
+
+在上面的图6中，FCP体验按用户的有效连接类型（ECT）分组。有意思的是，有效连接类型（ECT）速度与提供快速FCP的网站百分比间存在相关性。随着有效连接类型（ECT）速度的降低，快速体验的比例接近零。具有14％的站点为使用4G有效连接类型（ECT）的用户提供快速的FCP体验，而19％站点的体验较慢。为使用3G有效连接类型（ECT）的用户提供慢速FCP服务的网站比例为61％，为2G有效连接类型（ECT）的用户提供慢速FCP服务的网站比例为90％，为缓慢2G有效连接类型（ECT）的用户提供慢速FCP服务的网站比例为99％。这些结果表明，很少有网站对比4G慢的用户持续提供快速FCP服务。
+
+#### 按地理位置看FCP
+
+<figure id="fig7">
+  <a href="/static/images/2019/performance/fig7.png">
+    <img src="/static/images/2019/performance/fig7.png" alt="图7.标有快速，中速或慢速FCP的网站分布，按地理位置细分。" aria-labelledby="fig7-caption" aria-describedby="fig7-description" width="600" height="940" data-width="600" data-height="940" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=792398959&amp;format=interactive">
+  </a>
+  <div id="fig7-description" class="visually-hidden">排名前23位的最受欢迎地区的FCP分布条形图。韩国拥有最多快速的站点，为36％。从那里开始，其他地区的快速网站百分比迅速下降：日本28％，台湾26％，荷兰21％等。</div>
+  <figcaption id="fig7-caption">图7.标有快速，中速或慢速FCP的网站分布，按地理位置细分。</figcaption>
+</figure>
+
+最后，我们可以按用户的地理位置对FCP进行切片。上面的图表显示了拥有最多独立网站数量的前23个地理位置，这是开放网络总体流行程度的指标。美国的网络用户访问的独立网站最多，数量为1,211,002。地理位置按具有足够快速的FCP体验的网站百分比排序。排在首位的是三个[Asia-Pacific](https://en.wikipedia.org/wiki/Asia-Pacific) (APAC) 地理位置：韩国，台湾和日本。这可以通过在这些地区具有极高的[快速网络连接速度](https://en.wikipedia.org/wiki/List_of_countries_by_Internet_connection_speeds)。韩国有36％的网站符合快速FCP标准，只有7％被评为慢速FCP。回想一下，快速/中速/慢速网站的全球分布约为13/66/20，这使韩国区域成为一个明显积极的异常值。 
+
+其他亚太地区区域则讲述了一个不同的故事。泰国，越南，印度尼西亚和印度都拥有不到10％的快速网站。这些地理位置的慢速网站比例也比韩国高出三倍以上。
+
+### 首包时间(TTFB)
+
+[首包时间](https://web.dev/time-to-first-byte) (TTFB)是页面从用户导航到接收到响应的第一个字节为止花费时间的度量。
+
+<figure id="fig8">
+  <a href="/static/images/2019/performance/nav-timing.png">
+    <img alt="页面导航中Navigation Timing API的事件图" src="/static/images/2019/performance/nav-timing.png" alt="图8.页面导航中Navigation Timing API的事件图。" aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="2580" height="868">
+  </a>
+  <div id="fig8-description" class="visually-hidden">该图展示了页面加载过程的网络阶段：startTime (promptForUnload)，重定向，AppCache，DNS，TCP，请求，响应，处理，以及加载。</div>
+  <figcaption id="fig8-caption">图8.页面导航中Navigation Timing API的事件图。</figcaption>
+</figure>
+
+为了帮助解释TTFB及其影响因素，让我们借鉴一份[Navigation Timing API 规范](https://developer.mozilla.org/en-US/docs/Web/API/Navigation_timing_API)中的图表。在上面的图8中，TTFB是从`开始时间 startTime` 到 `响应时间 responseStart`，包括介于两者之间的所有内容：`卸载 unload`，`重定向 redirects`，`AppCache`，`DNS`，`SSL`，`TCP`，以及服务器处理请求的时间。在这些条件下，我们来看看这一指标用户的体验。
```

Comment:
Use local references if they exist:
```suggestion
为了帮助解释TTFB及其影响因素，让我们借鉴一份[Navigation Timing API 规范](https://developer.mozilla.org/zh-CN/docs/Web/API/Navigation_timing_API)中的图表。在上面的图8中，TTFB是从`开始时间 startTime` 到 `响应时间 responseStart`，包括介于两者之间的所有内容：`卸载 unload`，`重定向 redirects`，`AppCache`，`DNS`，`SSL`，`TCP`，以及服务器处理请求的时间。在这些条件下，我们来看看这一指标用户的体验。
```

Reply:
the local references didn't finish translation yet,so it's better stay in English version.